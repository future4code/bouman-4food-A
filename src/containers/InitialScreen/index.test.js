import React from "react";
import renderer from "react-test-renderer";
import { RectangleMain, mapDispatchToProps } from "./index"

describe("Testar o InitialScreen", ()=>{
    test("Snapshot", ()=>{
        const tree = renderer.create(<RectangleMain />).toJSON();
        expect(tree).toMatchSnapshot();
    })  
})

describe('mapDispatchToProps', () => {
    it('Testa se o mapDispatchToProps foi chamado', () => {
        const dispatch = jest.fn();
        mapDispatchToProps(dispatch).goToLoginPage();
        expect(dispatch).toHaveBeenCalledTimes(1);
    });
});

