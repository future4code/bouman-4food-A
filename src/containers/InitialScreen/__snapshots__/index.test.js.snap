// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar o InitialScreen Snapshot 1`] = `
<div
  className="sc-fYxtnH jebFPh"
/>
`;
